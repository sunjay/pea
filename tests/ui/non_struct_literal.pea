fn foo() {
    let x = foo { y: 3 };
    // Error should still be produced even if there are no field names
    let x = foo {};
}
