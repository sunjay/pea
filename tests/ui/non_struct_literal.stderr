[tests/ui/non_struct_literal.pea:2:19] error: cannot use initializer here since `foo` is not a struct or enum variant

[tests/ui/non_struct_literal.pea:2:13-2:15] error: `foo` is not a struct or enum variant

[tests/ui/non_struct_literal.pea:4:13-4:15] error: `foo` is not a struct or enum variant

error: aborting due to 3 previous errors

